railties/lib/rails/application/bootstrap.rb

```diff
@@ -35,7 +35,7 @@ module Bootstrap
       # Initialize the logger early in the stack in case we need to log some deprecation.
       initializer :initialize_logger, group: :all do
         Rails.logger ||= config.logger || begin
-          logger = ActiveSupport::Logger.new(config.default_log_file)
+          logger = ActiveSupport::Logger.new(config.default_log_file, 1, 100 * 1024 * 1024)
```

Comment:
I think we need to scope this to `development` and `test` environment. Some people may use a file logger in production with some `logrotate` daemon or something, we should absolutely not truncate their logs.

Maybe we should introduce another `config` entry to configure this, so that we can set it to a reasonable default in dev/test and no limit in production.

Reply:
Yeah makes sense, I believe `logrotate` is commonly used in `production` if one's wants to keep the default and avoid file bloats